### DISCUSSION TITLE
Is the license correct?

### DISCUSSION THREAD

**Discussion by mootari** [mkkellogg/GaussianSplats3D#96] [General]
Hi, I noticed that your source includes a couple of classes that appear to have been copied over or adapted from three.js, but [the license](https://github.com/mkkellogg/GaussianSplats3D/blob/3c05854e6d998e9e7e78991e724542232d708db1/LICENSE) makes no mention of them.

Could you clarify which code you inherited from three (e.g. by embedding [the three license](https://github.com/mrdoob/three.js/blob/d04539a76736ff500cae883d6a38b3dd8643c548/LICENSE) in those files)?

**Comment by mkkellogg** [comment]
Yeah `OrbitControls` was indeed mostly copied from Three.js, so I need to update that file with the license. I will do that in an upcoming release.

**Comment by mkkellogg** [comment]
Closing since I have added the license info to the relevant files.
